### ISSUE TITLE
[mwc-check-list-item] fails a11y test

### ISSUE THREAD

**Issue by neilvanlandingham** [material-components/material-web#4099]
`<mwc-check-list-item>` doesn't wrap `<mwc-checkbox>` with `<mwc-formfield>` or pass down the `aria-label` attribute, so the native `<input type="checkbox">`  has no associated label.

```
"failureSummary": "Fix any of the following:
+   Form element does not have an implicit (wrapped) <label>
+   Form element does not have an explicit <label>
+   aria-label attribute does not exist or is empty
+   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
+   Element has no title attribute
+   Element has no placeholder attribute
+   Element's default semantics were not overridden with role=\"none\" or role=\"presentation\"",
+         "html": "<input type=\"checkbox\" class=\"mdc-checkbox__native-control\" name=\"\" data-indeterminate=\"false\" value=\"on\">",
+         "impact": "critical",
```

**Comment by asyncLiz** [comment]
The MWC components no longer have resources to work on. Does this issue occur in the new M3 `<md-*` list components?

**Comment by asyncLiz** [comment]
Obsolete with M3
